setup.cfg

```diff
@@ -44,7 +44,7 @@ python_requires = >=3.6
 install_requires =
     execnet>=1.1
     pytest>=6.2.0
-    pytest-forked
+    py
```

Comment:
We still use `py` in pytest-xdist's own code, which is being fixed by #822, #823, and #824.